mathesar_ui/src/sections/table-view/display-options/FilterEntry.svelte

```diff
@@ -54,9 +56,11 @@
     <Select options={conditions} bind:value={condition}
       on:change={() => dispatch('reload')}/>
   </td>
-  <td class="value">
-    <TextInput bind:value={inputValue}/>
-  </td>
+  {#if !filterByDuplicates}
```

Comment:
I believe this change should be kept, as it keeps previous filter entries that chose get_duplicates as the operation consistent. Otherwise, the text box for the value will show up again.

Reply:
Sounds good!